index.php

```diff
@@ -20,4 +21,66 @@
  * to run a match use the method matchInDictionary
  * this will return an array of words and scores
  */
-var_dump($scrabble->matchInDictionary($rack));
+
+$words = $scrabble->matchInDictionary($rack);
+$split_letters = $scrabble->formatStringToArray($rack);
+
+?>
+
+<head>
+    <style><?php include 'style.css'; ?></style>
+</head>
+<body>
+<div class="container">
+    <div style="display: flex; flex-direction: column">
+        <div style="display: flex; flex-direction: column; align-items: center;">
+            <p>To change the letters in your rack simply replace them below and click update!</p>
+            <p>Your current rack: </p>
+            <?php echo "<form id='rackForm'>";
+            foreach ($split_letters as $index => $letter) {
+                echo "<input class='tile' value='$letter' type='text' maxlength='1' id='tile$index'/>";
+            } ?>

```

Comment:
I'm pretty big on FIG PSR and code readability, so I would have done (if forced to do inline):
```suggestion
            <?php 
                echo "<form id='rackForm'>";
                $inputTemplate = "<input class='%s' value='%s' type='%s' maxlength='%s' id='tile%s'/>";
                foreach ($split_letters as $index => $letter) {
                    echo sprintf (
                        $inputTemplate,
                        'tile',
                        $letter,
                        'text',
                        1,
                        $index
                    )
                } 
            ?>

```